tap_aggregator/src/main.rs

```diff
@@ -42,6 +47,26 @@ struct Args {
     /// Defaults to 5000.
     #[arg(long, default_value_t = 5000, env = "TAP_METRICS_PORT")]
     metrics_port: u16,
+
+    /// Domain name to be used for the EIP-712 domain separator.
+    #[arg(long, env = "TAP_DOMAIN_NAME")]
+    domain_name: Option<String>,
+
+    /// Domain version to be used for the EIP-712 domain separator.
+    #[arg(long, env = "TAP_DOMAIN_VERSION")]
+    domain_version: Option<String>,
+
+    /// Domain chain ID to be used for the EIP-712 domain separator.
+    #[arg(long, env = "TAP_DOMAIN_CHAIN_ID")]
+    domain_chain_id: Option<String>,
+
+    /// Domain verifying contract to be used for the EIP-712 domain separator.
+    #[arg(long, env = "TAP_DOMAIN_VERIFYING_CONTRACT")]
+    domain_verifying_contract: Option<Address>,
```

Comment:
Shouldn't these be mandatory arguments?

Reply:
Technically they are not required by eip712. Any or all of them could be skipped.

Reply:
Noted, thanks!